translate.js

```diff
@@ -0,0 +1,264 @@
+#! /usr/bin/node
```

Comment:
I'd suggest putting this into the `tools` folder

Reply:
understood. Will do.